api/token.go

```diff
@@ -538,6 +539,7 @@ func generateAccessToken(user *models.User, expiresIn time.Duration, secret stri
 		AppMetaData:  user.AppMetaData,
 		UserMetaData: user.UserMetaData,
 		Role:         user.Role,
+		SessionId:    sessionId,
```

Comment:
Can we also rename this `SessionID` as it appears to be the prevalent way of saying "ID" in the code.